client/hosting/overview/components/style.scss

```diff
@@ -212,27 +205,72 @@ $blueberry-color: #3858e9;
 
 	.plan-storage__bar .progress-bar__progress {
 		background-color: $blueberry-color;
-
 	}
 
 	.plan-storage__bar.is-alert .progress-bar__progress {
 		background-color: var(--studio-red-30);
 	}
 }
 
+.hosting-overview__site-metrics-footer {
+	display: flex;
+	justify-content: space-between;
+	gap: var(--grid-unit-10, 8px);
+	margin-top: 8px;
+}
+
+.hosting-overview__plan-storage,
+.hosting-overview__plan-bandwidth,
+.hosting-overview__plan-site-visits {
+	display: flex;
+	flex-direction: column;
+	border-radius: 4px;
+	border: 1px solid var(--studio-gray-0);
+	background: var(--studio-gray-0);
+	padding: var(--grid-unit-30, 24px);
```

Comment:
You're not wrong, it's kind of magical.

The `grid-unit` system feels familiar to me because I used it when working on Gutenberg... and when I saw it (with added fallbacks) in Calypso, I just kind of adapted and incorporated the fallbacks in the interest of consistency.

The grid units themselves are a [spacing system used in Core](https://github.com/WordPress/gutenberg/blob/5d408ce8849e6183cb39f48b071400b04a1cde78/packages/base-styles/_variables.scss#L25-L40). It's based on an 8px unit, named `--grid-unit-10`. From there, every ten `grid-units` is another 8px. That's why `grid-unit-30` equates to `24px`, though I agree those numbers feel awkward side by side. Seeing the math in the actual declarations I've linked above helped me wrap my brain around it when I started using it, and there's more context in [the proposal post](https://make.wordpress.org/design/2019/10/31/proposal-a-consistent-spacing-system-for-wordpress/) if you're curious 🙂 

From what I understand, I think the reason we're seeing `grid-unit`+`fallback` pattern in Calypso is to prepare for a potential future where we incorporate that system here. I'm not sure how organized of an effort that is, or if it's more just individual developers proactively future-proofing. I know there were [discussions about incorporating this within woo a while back](10045-gh-Automattic/woocommerce.com/files#diff-8a2ef6dca3124e9c64d765dd2e6663fd2f1543279068ae9ea9e3e1399aea3fee), but I wasn't able to find similar conversatons about calypso. Granted, the words "grid" "unit" and "Calypso" are utterly _horrible_ ingredients for a useful search term, so it's likely there's discussions/context out there I didn't come across in my brief search.

So, tl;dr: the `px` fallbacks are doing the real work here, but it looks like we're at least partially preparing for a day when a universal spacing system could be implemented, and putting the variables in place ahead of time.

Reply:
Outside `client/hosting/overview/components/style.scss`, I see only 2 instances of `--grid-unit-*` CSS variables being used in Calypso, which makes me question if we should really add them here. Is that correct, or am I missing something, @chad1008?

I absolutely see the appeal in having a system for spacing, but the low usage makes me think that the effort to implement this in Calypso might not be organized or it might have stalled.

Reply:
I have similar thought as Fredrik.
Additionally - `grid-unit`s doesn't work ATM, so there is possibility to make mistake as something will be broken as soon as they are integrated to Calypso. So anyway, it will be necessary to double-check all of them. And long story short, I would say - YAGNI.